build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/operator-framework/helm-operator:v0.7.0
+
+COPY watches.yaml ${HOME}/watches.yaml
+COPY helm-charts/ ${HOME}/helm-charts/
```

Comment:
Will it be good to download from actual charts repo

Reply:
Yes..can be done @AmitKumarDas . However, generally if the chart params is changed (esp. if new params are introduced or some old are removed), it should ideally prompt the creation of a new clusterServiceVersion CR. It may be possible that we automatically build with the stable charts directly (by downloading in the dockerfile) but use/refer to an older CSV in operatorhub. 

So, it may be better to make controlled changes to the helm charts in the repo - and add it as a release/post-release step similar to the way stable charts & rancher charts etc..,are updated. 

Having said that, still see the value of directly download actual charts. Can we defer this until a bit later when we figure out the exact cycles w/ the OLM.

Reply:
Yes. Ok.
We can download exact versions/releases as well.
I believe we will make mistakes/forget etc. if we dont have a single source of truth

Reply:
I agree, as part of the docker build - if we can pulling in the right version is a good option.

Reply:
May be we can put in a task to resolve these comments in the next PR.

Reply:
Created an issue for this here: https://github.com/openebs/helm-operator/issues/2

Reply:
Have taken the approach where helm charts will be pulled at build-time w/o us having to maintain them locally. The CI will generate the helm-operator image with the desired openebs_release_versions. 

The approach for new commits are discussed here: https://github.com/openebs/helm-operator/blob/70d47d2cfb11f08143ef86643d08e20187d0d4b9/CONTRIBUTING.md

Note: One of the reasons in not automating CSV creation is to ensure that the committer is aware of the changes made to stable charts (such as new components/updated values) and accordingly update the CSV artifact's alm-example.